src/components/forms/radioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styles from 'src/components/forms/radioButtonGroup/radioButtonGroup.module.css';
+import { RadioButton } from './components/RadioButton';
+import textStyles from "src/components/text/text.module.css";
+
+interface IOption {
+    id: string;
+    label: string;
+    disabled: boolean;
+}
+
+interface RenderOptionsProps {
+    options: IOption[];
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+interface RadioButtonGroupProps {
+    id: string;
+    label: string;
+    options: IOption[];
+    onValueChange: (name: string, value: string) => void;
```

Comment:
Normally, we'd set:
```suggestion
    onValueChange: (option: IOption) => void;
```
given that options are IOption[]